fastlane/Fastfile

```diff
@@ -477,13 +477,25 @@ platform :ios do
       download_dir: parent_dir_for_lprojs
     )
 
-    # Then redispatch the appropriate subset of translations back to the `InfoPlist.strings` and `Sites.strings` files in corresponding `*.lproj` dirs
-    parent_dir_for_intents_target_lprojs = File.join('WordPress', 'WordPressIntents')
+    git_commit(
+      path: File.join(parent_dir_for_lprojs, '*.lproj', 'Localizable.strings'),
+      message: 'Update app translations – `Localizable.strings`',
+      allow_nothing_to_commit: true
+    )
+
+    # Then redispatch the appropriate subset of translations back to the
+    # `InfoPlist.strings` and `Sites.strings` files in corresponding `*.lproj`
+    # dirs
     modified_files = ios_extract_keys_from_strings_files(
       source_parent_dir: parent_dir_for_lprojs,
       target_original_files: MANUALLY_MAINTAINED_STRINGS_FILES
     )
-    git_commit(path: modified_files, message: 'Update app translations', allow_nothing_to_commit: true)
+
+    git_commit(
+      path: modified_files,
+      message: 'Update app translations – Other `.strings`',
```

Comment:
I decided to use the same "root" for this commit as the one above and to differentiate them using "`Localizable.strings`" vs. "Other `.strings`".

There might be a nifty way to do it all in one commit, but I actually think it might be useful to do it separately. Apart from being the result of two different actions, it might also be useful to compare how a strings changed from GlotPress in the first commit with how it was extracted in the second.

Reply:
In #18200, there are no changes to the extracted `.strings`. I verified this works locally by editing a prefixed string in `Localizable.strings` and running the action.